packages/frontend/src/features/projectGroupAccess/components/ProjectGroupAccessModal.tsx

```diff
@@ -71,6 +76,45 @@ const ProjectGroupAccessModal: FC<ProjectGroupAccessModalProps> = ({
         },
     ] = useListState<ProjectGroupAccessState>();
 
+    const { data: projectAccesses, refetch: refetchProjectAccesses } =
+        useProjectGroupAccesses(projectUuid);
+    const { mutateAsync: addProjectGroupAccess } =
+        useAddProjectGroupAccessMutation();
+    const { mutateAsync: removeProjectGroupAccess } =
+        useRemoveProjectGroupAccessMutation();
+    useEffect(() => {
+        async function demo() {
+            if (groups === undefined) return;
+
+            const { data } = await refetchProjectAccesses();
+            console.log('initial project accesses', { data, projectAccesses });
+
+            const accessResponse = await addProjectGroupAccess({
+                projectUuid,
+                groupUuid: groups[0].uuid,
+                role: ProjectMemberRole.DEVELOPER,
+            });
+            console.log(
+                'response from adding project accesses',
+                accessResponse,
+            );
+
+            const { data: data2 } = await refetchProjectAccesses();
+            console.log('after adding project accesses', data2);
+
+            await removeProjectGroupAccess({
+                projectUuid,
+                groupUuid: groups[0].uuid,
+            });
+
+            const { data: data3 } = await refetchProjectAccesses();
+            console.log('after removing project accesses', data3);
+        }
+
+        demo();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [groups]);
```

Comment:
Do we want to remove this test code before merge?

Reply:
removed